src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -643,6 +643,13 @@ internal bool ContainsAnnotatedUnconstrainedTypeParameter()
 
         internal abstract TypeSymbol SetUnknownNullabilityForReferenceTypes();
 
+        /// <summary>
+        /// Returns a type with merged nested nullability, or false if nullability
+        /// cannot be merged. The caller is responsible for ensuring the two
+        /// types are identical, ignoring nullability.
+        /// </summary>
+        internal abstract (bool, TypeSymbol) TryMergeNullability(TypeSymbol other, VarianceKind variance);
```

Comment:
>(bool, TypeSymbol) [](start = 26, length = 18)

I think we should name the returned tuple, as that shows up in QuickInfo. Thanks

Reply:
As we discussed offline, I'll leave as is since this is a simple `Try` method where the meaning of the tuple elements should be clear.

---
In reply to: [224208051](https://github.com/dotnet/roslyn/pull/29727#discussion_r224208051) [](ancestors = 224208051)